src/sentry/rules/conditions/event_frequency.py

```diff
@@ -125,3 +127,75 @@ def query_hook(self, event, start, end, environment_id):
             environment_id=environment_id,
             use_cache=True,
         )[event.group_id]
+
+
+session_percent_intervals = {
+    "1m": ("one minute", timedelta(minutes=1)),
+    "5m": ("five minutes", timedelta(minutes=5)),
+    "10m": ("ten minutes", timedelta(minutes=10)),
+    "30m": ("30 minutes", timedelta(minutes=30)),
+    "1h": ("one hour", timedelta(minutes=60)),
+}
+
+
+class SessionPercentForm(EventFrequencyForm):
+    intervals = session_percent_intervals
+    interval = forms.ChoiceField(
+        choices=[
+            (key, label)
+            for key, (label, duration) in sorted(
+                session_percent_intervals.items(),
+                key=lambda key____label__duration: key____label__duration[1][1],
+            )
+        ]
+    )
+    value = forms.IntegerField(widget=forms.TextInput())
+
+
+class SessionPercentCondition(BaseEventFrequencyCondition):
```

Comment:
Wondering if this should be `EventFrequencyPercentCondition` instead. That way, when we migrate this over to use `metrics` it doesn't need to be renamed.